source/2019-11-01-the-ember-times-issue-122.md

```diff
@@ -42,13 +43,26 @@ As of now, it provides the `assert.contains` and `assert.matches` APIs for simpl
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Fine-Tune Validations with Ember Model Validator 3.10 🔍](https://twitter.com/esbanarango/status/1189759579493679106)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+[Ember Model Validator](https://github.com/esbanarango/ember-model-validator), which lets you define validations for Ember Data models, recently celebrated its 3.10 release! 🎉
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Thanks to [André Medeiros (@andsmedeiros)](https://github.com/andsmedeiros), you can now [pass `except` or `only` option to Ember Model Validator](https://github.com/esbanarango/ember-model-validator#usage) to run a subset of validations.
```

Comment:
I think it'd be

```suggestion
Thanks to [André Medeiros (@andsmedeiros)](https://github.com/andsmedeiros), you can now [pass the `except` or `only` option to Ember Model Validator](https://github.com/esbanarango/ember-model-validator#usage) to run a subset of validations.
```

Reply:
Yep, I was debating between the two. I do think the sentence flows better with "the" in-between.